server/repositories/cmsApi.js

```diff
@@ -1,37 +1,70 @@
-const { getEstablishmentUiId } = require('../utils');
+/* eslint-disable class-methods-use-this */
+const { DrupalJsonApiParams: Query } = require('drupal-jsonapi-params');
+
+const topicsQuery = new Query()
+  .addFields('taxonomy_term--tags', [
+    'drupal_internal__tid',
+    'name',
+    'description',
+  ])
+  .addFields('taxonomy_term--moj_categories', [
+    'name',
+    'description',
+    'field_legacy_landing_page',
+  ])
+  .addFilter(
+    'vid.meta.drupal_internal__target_id',
+    ['moj_categories', 'tags'],
+    'IN',
+  )
+  .addPageLimit(100)
+  .getQueryString();
 
 class CmsApi {
   constructor(jsonApiClient) {
     this.jsonApiClient = jsonApiClient;
   }
 
-  async primaryMenu(prisonId = 0) {
-    const response = await this.jsonApiClient.get(
-      `/jsonapi/node/landing_page?fields[node--landing_page]=title,field_moj_description,drupal_internal__nid,field_moj_prisons`,
+  async getTopics(establishmentName) {
+    if (!establishmentName) {
+      return [];
+    }
```

Comment:
think this might be a bit lame - could potentially ensure that we have a prison in context on the frontend and throw if not.

Reply:
Pushed error handling of missing prison up to controller to fail hard as its more observable